src/ResourceManager/Insights/Commands.Insights/Autoscale/RemoveAzureRmAutoscaleSettingCommand.cs

```diff
@@ -14,13 +14,14 @@
 
 using Microsoft.Azure.Management.Insights;
 using System.Management.Automation;
+using Microsoft.Azure.Management.Insights.Models;
 
 namespace Microsoft.Azure.Commands.Insights.Autoscale
 {
     /// <summary>
     /// Remove an autoscale setting.
     /// </summary>
-    [Cmdlet(VerbsCommon.Remove, "AzureRmAutoscaleSetting"), OutputType(typeof(AzureOperationResponse))]
+    [Cmdlet(VerbsCommon.Remove, "AzureRmAutoscaleSetting"), OutputType(typeof(PSObject))]
```

Comment:
The new spec from Swagger is not returning anything for Delete operations. The Hydra spec (old one) used to return an object with request Id and status code.